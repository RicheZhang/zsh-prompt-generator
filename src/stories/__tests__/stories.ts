import initStoryshots from '@storybook/addon-storyshots';
// `require.context()` in .storybook/config.ts is originally Webpack's func.
// It enables `require.context()` for babel.
import registerRequireContextHook from 'babel-plugin-require-context-hook/register';
registerRequireContextHook();

initStoryshots();
