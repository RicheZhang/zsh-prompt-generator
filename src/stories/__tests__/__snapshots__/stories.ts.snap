// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Storyshots RoundedButton with some emoji 1`] = `
<div>
  <div>
    <div
      style={
        Object {
          "backgroundColor": "#fff",
          "border": "1px solid #eee",
          "borderRadius": "2px",
          "color": "black",
          "fontFamily": "Helvetica Neue, Helvetica, Segoe UI, Arial, freesans, sans-serif",
          "fontSize": "15px",
          "fontWeight": 300,
          "lineHeight": 1.45,
          "marginBottom": "20px",
          "marginTop": "20px",
          "padding": "20px 40px 40px",
        }
      }
    >
      <div
        style={
          Object {
            "borderBottom": "1px solid #eee",
            "marginBottom": 10,
            "paddingTop": 10,
          }
        }
      >
        <h1
          style={
            Object {
              "fontSize": "35px",
              "margin": 0,
              "padding": 0,
            }
          }
        >
          RoundedButton
        </h1>
        <h2
          style={
            Object {
              "fontSize": "22px",
              "fontWeight": 400,
              "margin": "0 0 10px 0",
              "padding": 0,
            }
          }
        >
          with some emoji
        </h2>
      </div>
    </div>
  </div>
  <div
    id="story-root"
    style={Object {}}
  >
    <button
      onClick={[Function]}
      style={
        Object {
          "backgroundColor": "papayawhip",
          "border": "none",
          "borderRadius": 5,
          "color": "white",
          "display": "inline-block",
          "fontSize": "16px",
          "margin": "4px 2px",
          "padding": 20,
          "textAlign": "center",
          "textDecoration": "none",
        }
      }
    >
      <span
        aria-label="so cool"
        role="img"
      >
        üòÄ üòé üëç üíØ
      </span>
    </button>
  </div>
  <div>
    <div
      style={
        Object {
          "backgroundColor": "#fff",
          "border": "1px solid #eee",
          "borderRadius": "2px",
          "color": "black",
          "fontFamily": "Helvetica Neue, Helvetica, Segoe UI, Arial, freesans, sans-serif",
          "fontSize": "15px",
          "fontWeight": 300,
          "lineHeight": 1.45,
          "marginBottom": "20px",
          "marginTop": "20px",
          "padding": "20px 40px 40px",
        }
      }
    >
      
      <div>
        <h1
          style={
            Object {
              "borderBottom": "1px solid #EEE",
              "fontSize": "25px",
              "margin": "20px 0 0 0",
              "padding": "0 0 5px 0",
            }
          }
        >
          Story Source
        </h1>
        <pre
          style={
            Object {
              "alignItems": "center",
              "backgroundColor": "#fafafa",
              "display": "flex",
              "fontFamily": "Menlo, Monaco, \\"Courier New\\", monospace",
              "fontSize": ".88em",
              "justifyContent": "space-between",
              "lineHeight": 1.5,
              "overflowX": "scroll",
              "padding": ".5rem",
            }
          }
        >
          <div>
            <div>
              <div
                style={
                  Object {
                    "paddingLeft": 18,
                    "paddingRight": 3,
                  }
                }
              >
                <span
                  style={
                    Object {
                      "color": "#444",
                    }
                  }
                >
                  &lt;
                  RoundedButton
                </span>
                <span>
                  <span>
                     
                    <span
                      style={Object {}}
                    >
                      color
                    </span>
                    <span>
                      =
                      <span
                        style={Object {}}
                      >
                        "
                        <span
                          style={
                            Object {
                              "color": "#22a",
                              "wordBreak": "break-word",
                            }
                          }
                        >
                          papayawhip
                        </span>
                        "
                      </span>
                    </span>
                  </span>
                  <span>
                     
                    <span
                      style={Object {}}
                    >
                      onClick
                    </span>
                    <span>
                      =
                      <span
                        style={Object {}}
                      >
                        {
                        <span
                          style={
                            Object {
                              "color": "#170",
                            }
                          }
                        >
                          actionHandler
                        </span>
                        }
                      </span>
                    </span>
                    
                  </span>
                </span>
                <span
                  style={
                    Object {
                      "color": "#444",
                    }
                  }
                >
                  &gt;
                </span>
              </div>
              <div>
                <div
                  style={
                    Object {
                      "paddingLeft": 33,
                      "paddingRight": 3,
                    }
                  }
                >
                  <span
                    style={
                      Object {
                        "color": "#444",
                      }
                    }
                  >
                    &lt;
                    span
                  </span>
                  <span>
                    <span>
                       
                      <span
                        style={Object {}}
                      >
                        role
                      </span>
                      <span>
                        =
                        <span
                          style={Object {}}
                        >
                          "
                          <span
                            style={
                              Object {
                                "color": "#22a",
                                "wordBreak": "break-word",
                              }
                            }
                          >
                            img
                          </span>
                          "
                        </span>
                      </span>
                    </span>
                    <span>
                       
                      <span
                        style={Object {}}
                      >
                        aria-label
                      </span>
                      <span>
                        =
                        <span
                          style={Object {}}
                        >
                          "
                          <span
                            style={
                              Object {
                                "color": "#22a",
                                "wordBreak": "break-word",
                              }
                            }
                          >
                            so cool
                          </span>
                          "
                        </span>
                      </span>
                      
                    </span>
                  </span>
                  <span
                    style={
                      Object {
                        "color": "#444",
                      }
                    }
                  >
                    &gt;
                  </span>
                </div>
                <div
                  style={
                    Object {
                      "paddingLeft": 48,
                      "paddingRight": 3,
                    }
                  }
                >
                  <span
                    style={
                      Object {
                        "color": "#444",
                      }
                    }
                  >
                    üòÄ üòé üëç üíØ
                  </span>
                </div>
                <div
                  style={
                    Object {
                      "paddingLeft": 33,
                      "paddingRight": 3,
                    }
                  }
                >
                  <span
                    style={
                      Object {
                        "color": "#444",
                      }
                    }
                  >
                    &lt;/
                    span
                    &gt;
                  </span>
                </div>
              </div>
              <div
                style={
                  Object {
                    "paddingLeft": 18,
                    "paddingRight": 3,
                  }
                }
              >
                <span
                  style={
                    Object {
                      "color": "#444",
                    }
                  }
                >
                  &lt;/
                  RoundedButton
                  &gt;
                </span>
              </div>
            </div>
          </div>
          <button
            onClick={[Function]}
            style={
              Object {
                "alignSelf": "flex-start",
                "backgroundColor": "rgb(255, 255, 255)",
                "borderColor": "rgb(238, 238, 238)",
                "borderImage": "initial",
                "borderRadius": "3px",
                "borderStyle": "solid",
                "borderWidth": "1px",
                "cursor": "pointer",
                "flexShrink": "0",
                "fontSize": "13px",
                "overflow": "hidden",
                "padding": "3px 10px",
              }
            }
            type="button"
          >
            Copy
          </button>
        </pre>
      </div>
      <div>
        <h1
          style={
            Object {
              "borderBottom": "1px solid #EEE",
              "fontSize": "25px",
              "margin": "20px 0 0 0",
              "padding": "0 0 5px 0",
            }
          }
        >
          Prop Types
        </h1>
        <div>
          <h3
            style={
              Object {
                "margin": "20px 0 0 0",
              }
            }
          >
            "
            RoundedButton
            " Component
          </h3>
          <small>
            No propTypes defined!
          </small>
        </div>
      </div>
    </div>
  </div>
</div>
`;

exports[`Storyshots RoundedButton with text 1`] = `
<div>
  <div>
    <div
      style={
        Object {
          "backgroundColor": "#fff",
          "border": "1px solid #eee",
          "borderRadius": "2px",
          "color": "black",
          "fontFamily": "Helvetica Neue, Helvetica, Segoe UI, Arial, freesans, sans-serif",
          "fontSize": "15px",
          "fontWeight": 300,
          "lineHeight": 1.45,
          "marginBottom": "20px",
          "marginTop": "20px",
          "padding": "20px 40px 40px",
        }
      }
    >
      <div
        style={
          Object {
            "borderBottom": "1px solid #eee",
            "marginBottom": 10,
            "paddingTop": 10,
          }
        }
      >
        <h1
          style={
            Object {
              "fontSize": "35px",
              "margin": 0,
              "padding": 0,
            }
          }
        >
          RoundedButton
        </h1>
        <h2
          style={
            Object {
              "fontSize": "22px",
              "fontWeight": 400,
              "margin": "0 0 10px 0",
              "padding": 0,
            }
          }
        >
          with text
        </h2>
      </div>
    </div>
  </div>
  <div
    id="story-root"
    style={Object {}}
  >
    <button
      onClick={[Function]}
      style={
        Object {
          "backgroundColor": "hotpink",
          "border": "none",
          "borderRadius": 5,
          "color": "white",
          "display": "inline-block",
          "fontSize": "16px",
          "margin": "4px 2px",
          "padding": 20,
          "textAlign": "center",
          "textDecoration": "none",
        }
      }
    >
      Hello Button
    </button>
  </div>
  <div>
    <div
      style={
        Object {
          "backgroundColor": "#fff",
          "border": "1px solid #eee",
          "borderRadius": "2px",
          "color": "black",
          "fontFamily": "Helvetica Neue, Helvetica, Segoe UI, Arial, freesans, sans-serif",
          "fontSize": "15px",
          "fontWeight": 300,
          "lineHeight": 1.45,
          "marginBottom": "20px",
          "marginTop": "20px",
          "padding": "20px 40px 40px",
        }
      }
    >
      
      <div>
        <h1
          style={
            Object {
              "borderBottom": "1px solid #EEE",
              "fontSize": "25px",
              "margin": "20px 0 0 0",
              "padding": "0 0 5px 0",
            }
          }
        >
          Story Source
        </h1>
        <pre
          style={
            Object {
              "alignItems": "center",
              "backgroundColor": "#fafafa",
              "display": "flex",
              "fontFamily": "Menlo, Monaco, \\"Courier New\\", monospace",
              "fontSize": ".88em",
              "justifyContent": "space-between",
              "lineHeight": 1.5,
              "overflowX": "scroll",
              "padding": ".5rem",
            }
          }
        >
          <div>
            <div>
              <div
                style={
                  Object {
                    "paddingLeft": 18,
                    "paddingRight": 3,
                  }
                }
              >
                <span
                  style={
                    Object {
                      "color": "#444",
                    }
                  }
                >
                  &lt;
                  RoundedButton
                </span>
                <span>
                  <span>
                     
                    <span
                      style={Object {}}
                    >
                      color
                    </span>
                    <span>
                      =
                      <span
                        style={Object {}}
                      >
                        "
                        <span
                          style={
                            Object {
                              "color": "#22a",
                              "wordBreak": "break-word",
                            }
                          }
                        >
                          hotpink
                        </span>
                        "
                      </span>
                    </span>
                  </span>
                  <span>
                     
                    <span
                      style={Object {}}
                    >
                      onClick
                    </span>
                    <span>
                      =
                      <span
                        style={Object {}}
                      >
                        {
                        <span
                          style={
                            Object {
                              "color": "#170",
                            }
                          }
                        >
                          actionHandler
                        </span>
                        }
                      </span>
                    </span>
                    
                  </span>
                </span>
                <span
                  style={
                    Object {
                      "color": "#444",
                    }
                  }
                >
                  &gt;
                </span>
              </div>
              <div
                style={
                  Object {
                    "paddingLeft": 33,
                    "paddingRight": 3,
                  }
                }
              >
                <span
                  style={
                    Object {
                      "color": "#444",
                    }
                  }
                >
                  Hello Button
                </span>
              </div>
              <div
                style={
                  Object {
                    "paddingLeft": 18,
                    "paddingRight": 3,
                  }
                }
              >
                <span
                  style={
                    Object {
                      "color": "#444",
                    }
                  }
                >
                  &lt;/
                  RoundedButton
                  &gt;
                </span>
              </div>
            </div>
          </div>
          <button
            onClick={[Function]}
            style={
              Object {
                "alignSelf": "flex-start",
                "backgroundColor": "rgb(255, 255, 255)",
                "borderColor": "rgb(238, 238, 238)",
                "borderImage": "initial",
                "borderRadius": "3px",
                "borderStyle": "solid",
                "borderWidth": "1px",
                "cursor": "pointer",
                "flexShrink": "0",
                "fontSize": "13px",
                "overflow": "hidden",
                "padding": "3px 10px",
              }
            }
            type="button"
          >
            Copy
          </button>
        </pre>
      </div>
      <div>
        <h1
          style={
            Object {
              "borderBottom": "1px solid #EEE",
              "fontSize": "25px",
              "margin": "20px 0 0 0",
              "padding": "0 0 5px 0",
            }
          }
        >
          Prop Types
        </h1>
        <div>
          <h3
            style={
              Object {
                "margin": "20px 0 0 0",
              }
            }
          >
            "
            RoundedButton
            " Component
          </h3>
          <small>
            No propTypes defined!
          </small>
        </div>
      </div>
    </div>
  </div>
</div>
`;
